src/Language/Dutch.php

```diff
@@ -0,0 +1,423 @@
+        "voor de",
+        "auf",
+        "dal",
+        "don",
+        "onder ’t",
+        "ten",
+        "van ’t",
+        "Voor den",
+        "Auf dem",
+        "Dal’",
+        "Dos",
+        "Onder de",
+        "Ter",
+        "Van de",
+        "voor den",
+        "auf dem",
+        "dal’",
+        "dos",
+        "onder de",
+        "ter",
+        "van De",
+        "Voor in ’t",
+        "Auf den",
+        "Dalla",
+        "Du",
+        "Onder den",
+        "Tho",
+        "van de",
+        "voor in ’t",
+        "auf den",
+        "dalla",
+        "du",
+        "onder den",
+        "tho",
+        "Van de l",
+        "Voor in t",
+        "Auf der",
+        "Das",
+        "El",
+        "Onder het",
+        "Thoe",
+        "van de l",
+        "voor in t",
+        "auf der",
+        "das",
+        "el",
+        "onder het",
+        "thoe",
+        "Van de l’",
+        "Vor",
+        "Auf ter",
+        "De",
+        "Het",
+        "Onder t",
+        "Thor",
+        "van de l’",
+        "vor",
+        "auf ter",
+        "de",
+        "het",
+        "onder t",
+        "thor",
+        "Van Den",
+        "Vor der",
+        "Aus",
+        "De die",
+        "I",
+        "Op",
+        "To",
+        "Van den",
+        "vor der",
+        "aus",
+        "de die",
+        "i",
+        "op",
+        "to",
+        "van den",
+        "Zu",
+        "Aus ’m",
+        "De die le",
+        "Im",
+        "Op ’t",
+        "Toe",
+        "Van Der",
+        "zu",
+        "aus ’m",
+        "de die le",
+        "im",
+        "op ’t",
+        "toe",
+        "Van der",
+        "Zum",
+        "Aus dem",
+        "De l",
+        "In",
+        "Op de",
+        "Tot",
+        "van der",
+        "zum",
+        "aus dem",
+        "de l",
+        "in",
+        "op de",
+        "tot",
+        "Van gen",
+        "Zur",
+        "Aus den",
+        "De l’",
+        "In ’t",
+        "Op den",
+        "Uijt",
+        "van gen",
+        "zur",
+        "aus den",
+        "de l’",
+        "in ’t",
+        "op den",
+        "uijt",
+        "Van het",
+        "Aus der",
+        "De la",
+        "In de",
+        "Op der",
+        "Uijt ’t",
+        "van het",
+        "aus der",
+        "de la",
+        "in de",
+        "op der",
+        "uijt ’t",
+        "Van la",
+    ];
+
+    public function getSuffixes(): array
+    {
+        return self::SUFFIXES;
+    }
+
+    public function getSalutations(): array
+    {
+        return array_combine(self::SALUTATIONS, self::SALUTATIONS);
+    }
+
+    public function getLastnamePrefixes(): array
+    {
+        $prefixMap = [];
+        foreach(self::LASTNAME_PREFIXES as $prefix){
+            $key = strtolower($prefix);
+//            $key = str_replace(' ', '-', $key);
+//            $key = str_replace('’', '', $key);
+            $prefixMap[$key] = $prefix;
+
+            // add version with ' instead of ’
```

Comment:
We prefer to let the code speak for itself unless a comment is required to explain the background/reasoning of the implementation. I would suggest to drop this comment.